### ISSUE TITLE
backend errors (again) when database is starting the first time

### ISSUE THREAD

**Issue by Baschdl** [match4everyone/match4everything#57]
Following error occurs, when deleting `database/data` and starting for the first time:
```
django.db.utils.OperationalError: could not connect to server: Connection refused
	Is the server running on host "database" (172.21.0.2) and accepting
	TCP/IP connections on port 5432?
``

**Comment by bjrne** [comment]
This only happens with docker I suppose?

**Comment by Baschdl** [comment]
I did not test it with a local postgres instance but for our project you could say that it "only happens with docker".
The cause is that something with the "wait for db" logic doesn't work @kevihiiin

**Comment by kevihiiin** [comment]
The logic is changed again in PR #21, I would wait until that PR has been merged and if the error still persists, I will have another look at the waiting part

**Comment by maltezacharias** [comment]
Adressed in c458215845d9ef1044df59e70c68ec764c07bab8, will be closed by #21
